docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.        chunksize = next_pow2(m) 
</pre><p>In contrast to the algorithm proposed in the paper, the MATLAB code uses <code>next_pow2(8 * m)</code></p><p> </p><p>Why <code>8</code> ? It has a code smell. Also, in another part of the code, we see:</p><pre class="ql-syntax" spellcheck="false">#from MATLAB
addParameter(p,'lookahead',2^nextpow2(16*SubsequenceLength));
</pre><p>which seems to be for <strong>forward</strong> processing. But, why <code>16</code>?  </p><p> </p><p>MAYBE the authors wanted to make sure that the algorithm can still consider a good number of subsequences in the first round even when the subsequence length m is [close to] a power of two. For instance, if <code>m == 32</code> ,  <code>next_pow2(m)</code> returns 32.  In this case, the first chunksize will be 32, meaning that it contains only one subsequence if exclusion zone is set to <code>m</code> </p><p> </p><p>I prefer to avoid using 8 or 16 for now. [We can later test and see how much it improves the performance in edge cases.]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Agreed (re: code smell). </p>

Reply:
Cool! For now, I am going to leave it as is and resolve this conversation. I will add a note to the comment  at the top of this PR so that we do not forget about it.